### ISSUE TITLE
inconsistent window dragging

### ISSUE THREAD

**Issue by talonrenaud** [gmarull/qtmodern#50]
When dragging the window using the title bar the event is ignored and the window doesn't move.
It randomly works after a few tries. (using Jerakin:borderless fork Commit #[4bd2122477a32e7348b5714ac8415fc72f07b2f3](https://github.com/Jerakin/qtmodern/commit/4bd2122477a32e7348b5714ac8415fc72f07b2f3)

Using PySide2 5.13.2 + Windows 10

![window_drag_issue](https://user-images.githubusercontent.com/6299536/74805479-0b851880-5298-11ea-8af8-032cfca256c9.gif)

**Comment by talonrenaud** [comment]
It looks like when I click and immediately drag the window, the window doesn't move.
Holding down the mouse click for a second or more and then dragging works.

**Comment by Jerakin** [comment]
I am can't really reproduce this, could it be because you are sometimes clicking "outside" of the window titlebar? The titlebar is currently only about 60% of a "standard" windows titlebar.

**Comment by talonrenaud** [comment]
I checked many times to make sure i was clicking on the smaller than usual titlebar but I get the same random drag behavior.
I tested on a completely different machine to be sure and I get the same behavior. Are you guys using PySide2 or PyQt5 for testing ?

**Comment by Jerakin** [comment]
PyQt5 is the bindings I use. I tried on Window 10 running in a VirtualBox. Will try it on a proper windows machine this weekend.

Does the master branch behave the same?

**Comment by talonrenaud** [comment]
Maybe I was clicking outside of the small title bar area. I made some changes to make the the drag-able title bat the same height as the min/max/close buttons and it makes it easier to drag now.
I also made small changes so that the title bar background color can be customized if needed which makes it even easier to locate the area that can be dragged.
This can be closed. Thanks again for this great project.
